src/main/java/com/exasol/adapter/dialects/redshift/RedshiftSqlDialect.java

```diff
@@ -123,7 +124,8 @@ public String getStringLiteral(final String value) {
             return "NULL";
         } else {
             if (value.contains("'") || value.contains("\\")) {
-                throw new IllegalArgumentException("Redshift string literal contains illegal characters: ' or \\.");
+                throw new IllegalArgumentException(ExaError.messageBuilder("E-VS-RS-1")
```

Comment:
I would use REDSH or RDSH instead of RS, because it gives more hint which VS it is